src/core/utils/otns.cpp

```diff
@@ -133,4 +134,9 @@ void Otns::EmitNeighborChange(otNeighborTableEvent aEvent, Neighbor &aNeighbor)
 } // namespace Utils
 } // namespace ot
 
+OT_TOOL_WEAK void otPlatOtnsStatus(const char *aStatus)
+{
+    _otLogFormatter(OT_LOG_LEVEL_NONE, OT_LOG_REGION_CORE, "[OTNS] %s", aStatus, NULL);
```

Comment:
I would advice against using `_otLogFormatter()` from another file.

All macors starting with prefix `_` should be considered private to the header file itself and not used by anyone else.

Reply:
If we need specific logging behavior (e.g. logging with `[OTNS]` or at certain level) for `otns`, I suggest adding specific new macros for that in `logging.hpp` directly.

Reply:
Ok, that makes sense.